utils/src/main/java/org/apache/cloudstack/utils/log/Logger.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.cloudstack.utils.log;
+
+/**
+ * wrapper class for the logging functionality of the log library in use
+ */
+public class Logger {
+    private org.apache.log4j.Logger logger;
+    public Logger(org.apache.log4j.Logger logger) {
+        this.logger = logger;
+    }
+
+    public void warn(String s, Exception e) {
+        logger.warn(s,e);
+    }
+
+    public void info(String s) {
+        logger.info(s);
+    }
+
+    public void error(String s) {
```

Comment:
Can you add a`public void error (String msg, Throwable e)` and change the `public void warn(String s, Exception e)` to `public void warn(String s, Throwable e)`

Reply:
I will as we progress change the interface to what we really use. I am also still considering to use an actual java-interface. Unfortunately java requires an explicit 'implements', otherwise the proxy mechs would be some easier.